utils/src/ast-grep/package-json.test.ts

```diff
@@ -0,0 +1,175 @@
+import assert from "node:assert/strict";
+import { describe, it } from "node:test";
+import dedent from "dedent";
+import jsonLang from "@ast-grep/lang-json";
+import { registerDynamicLanguage, parse } from "@ast-grep/napi";
+import type { Edit } from "@ast-grep/napi";
+import { getScriptsNode, getNodeJsUsage, replaceNodeJsArgs } from "./package-json.ts";
+
+registerDynamicLanguage({ json: jsonLang });
+
+describe("package-json utilities", () => {
+	describe("getScriptsNode", () => {
+		it("should get the scripts node", () => {
+			const input = dedent`
+				{
+					"scripts": {
+						"test": "echo \\"Error: no test specified\\" && exit 1"
+					}
+				}
+			`;
+
+			const result = getScriptsNode(parse('json', input));
+
+			assert(result);
+			assert.strictEqual(result.children().length, 3); // curly braces + pair + curly braces
```

Comment:
If suggestion on the way `getScriptsNode` works is accepted, it need to be updated to
```suggestion
			assert.strictEqual(result.length, 1); // number of scripts
```